packages/plugin-websocket/src/WebSocket.js

```diff
@@ -80,7 +80,7 @@ export default class WebSocket {
   }
 
   _connect() {
-    this._socket = new WebSocket(this._config.url);
+    this._socket = Reflect.construct(window.WebSocket, [this._config.url]);
```

Comment:
Nemelo by se tady pouzit spis `this._window.getWindow().WebSocket`? Prakticky to nic nezmeni, jelikoz se tohle nikdy nepouzije na serveru, akorat mam za to, ze se k window ma vseobecne v ime pristupovat timto zpusobem.